mergify_engine/actions/merge/helpers.py

```diff
@@ -73,19 +73,22 @@ def merge_report(ctxt, strict):
     return conclusion, title, summary
 
 
-def get_wait_for_ci_report(pull):
-    queue_status = ""
+def get_wait_for_ci_report(pull, rule=None, missing_conditions=None):
+    summary = "The pull request has been automatically updated to follow its base branch and will be merged soon."
+
     pulls = queue.get_pulls_from_queue(pull)
     if pulls:
         links = ", ".join((f"#{pull}" for pull in pulls))
-        queue_status = f"The following pull requests are queued: {links}"
+        summary += f"\n\nThe following pull requests are queued: {links}"
+
+    if rule and missing_conditions is not None:
+        summary += "\n\nThe required statuses are:\n"
+        for cond in rule["conditions"]:
+            if cond.attribute_name.startswith("status-"):
+                checked = " " if cond in missing_conditions else "X"
+                summary += f"\n- [{checked}] `{cond}`"
```

Comment:
Why only reporting the CI conditions?
I imagine that in 99% of the case it's only going to be `status-*` that are missing, but I wonder if it's a good idea to special-case that anyhow?

Reply:
If they is something, the check go to state cancelled instead of pending.
I think it's a good idea to put everything, so in case of cancellation, they will see why.